lib/queuePopulator/IngestionReader.js

```diff
@@ -71,22 +71,35 @@ class IngestionReader extends LogReader {
     }
 
     setup(done) {
-        this._setupIngestionProducer(err => {
-            if (err) {
-                return done(err);
-            }
-            return this._iProducer.getRaftId(this.bucket, (err, data) => {
+        const retryableError = { retryable: true };
+        async.series([
+            next => this._setupIngestionProducer(next),
```

Comment:
could use feedback on the idea of retryable errors

If an error occurs from `setupIngestionProducer`, this would mean the error occurred from mongo itself, so I decided this was an error that should crash pod as it can be a service down

Reply:
updated to log error and avoid crashing pod